src/content/docs/arcitect/installation.mdx

```diff
@@ -83,78 +83,46 @@ This enables communication with the [DataHUB](/nfdi4plants.knowledgebase/datahub
 
 <TabItem label="Linux" icon='linux'>
 
-:::note[Git Installation]
-Please install <a href="https://git-scm.com/downloads" target="_blank">Git</a> and <a href="https://git-lfs.com" target="_blank">Git LFS</a> before installing the ARCitect.
-:::
-
-:::tip
-For Linux we recommend to install via homebrew as described on the site above.
-:::
+### Git Installation
 
-<Steps>
+<GitDownloadInstall />
 
-1. Head to the ARCitect's GitHub <a href="https://github.com/nfdi4plants/ARCitect/releases/latest" target="_blank">release page</a>.
-2. Download the Linux AppImage `ARCitect-x.x.x.AppImage`.
-3. (optional) If you run Ubuntu 22.04 you need to download an additional library. Ubuntu 22.04 can't run AppImage files without the not natively installed FUSE library. To fix this, install the necessary library. To do this you need root permissions.
+### ARCitect Installation
 
-    ```bash title="Requires root permission"
-    sudo apt install libfuse2
-    ```
+:::note[Installation now based on DEB package]
+To avoid issues on certain Linux distributions, ARCitect installation is based on DEB package (instead of AppImage) since version 0.0.59. 
+:::
 
-4. Make the ARCitect executable.
+<Steps>
 
-    You need to make the downloaded AppImage executable before you can run it.
-    You can use the user interface or the command line to do this.
+1. Install additional dependencies via
     
-    <Tabs>
-
-    <TabItem label="User Interface">
-    <Steps>
-    1. Open the folder where the AppImage is currently saved (likely in Downloads).
-    2. (optional) Move the file to a location where you want it saved.
-    3. Right-click the file and select properties.
-
-        ![](@images/arcitect/linux-rightclick.png)
-
-    4. Go to the "permissions" tab and tick the field "execute".
-
-        ![](@images/arcitect/linux-permissions.png)
-
-    5. Now double-clicking the AppImage should start the ARCitect.
-
-        ![](@images/arcitect/linux-checkbox.png)
-
-    </Steps>
-    </TabItem>
-
-    <TabItem label="Command Line">
-    <Steps>
-
-    1. Open a terminal.
-    2. Move to the directory where you have saved your AppImage (e.g. Downloads).
+    ```bash
+    sudo apt-get update
+    sudo apt-get install libgtk-3-0 libnotify4 xdg-utils libsecret-1-0
+    ```
```

Comment:
You may need to run `sudo apt --fix-broken install` afterwards. We tested it on two machines, one debian and one ubuntu. Debian required this command to successfully install the package.